### PR TITLE
Remove SatInfo request

### PR DESCRIPTION
**Pull request by haslinghuis**
See: https://github.com/betaflight/betaflight/pull/13244

### CODE REVIEW

**Review by ctzsnooze — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by asizon — APPROVED** [review]

**Pull request merged**
